src/Components/SystemDetail/SystemRecommendations.js

```diff
@@ -24,26 +25,89 @@ const RecommendationsTable = asyncComponent(() => import('./RecommendationsTable
  *
  * @param props the props given by the smart component.
  */
+const defaultFilters = {
+    name: {
+        label: 'Name',
+        value: ''
+    }
+};
+
+export const isEmptyFilters = (activeFilters) => Object.values(activeFilters).find(
+    (item) => item?.value?.length > 0 || item?.length > 0
+);
+
+export const constructActiveFilters = (activeFilters) => Object.entries(activeFilters).map(([key, { label, value } = {}]) => ({
+    category: label,
+    chipKey: key,
+    chips:
+        value?.length > 0
+            ? Array.isArray(value)
+                ? value.map((item) => ({ name: item })) : [{ name: value }]
+            : []
+}));
+
+export const onDeleteFilter = (activeFilters, itemsToRemove) => {
```

Comment:
There is also 3rd argument `isAll` to help you when removing all filters.

Reply:
Yes, I am handing it [here](https://github.com/RedHatInsights/ros-frontend/pull/28/files#diff-a9708a2e840103f3e7638ad7a8489523b55d88a43edcfc9de456fbba4c67573bR158). 
Do you want me to modify logic so that `onDeleteFilter` will return `defaultFilters` when `isAll`?

Reply:
Oooh, no need to! This is enough.